libs/spark/src/Unstable_Tag/Unstable_Tag.stories.tsx

```diff
@@ -0,0 +1,251 @@
+import * as React from 'react';
+import type { Meta, Story } from '@storybook/react/types-6-0';
+import { Unstable_Tag, unstable_createSvgIcon } from '..';
+import { sparkThemeProvider } from '../../stories';
+import { Unstable_TagProps } from './Unstable_Tag';
+
+export const SbUnstable_Tag = Unstable_Tag;
+
+const Filter = unstable_createSvgIcon(
+  <path d="M2.25 6A.75.75 0 0 1 3 5.25h18a.75.75 0 0 1 0 1.5H3A.75.75 0 0 1 2.25 6Zm3 6a.75.75 0 0 1 .75-.75h12a.75.75 0 0 1 0 1.5H6a.75.75 0 0 1-.75-.75ZM10 17.25a.75.75 0 0 0 0 1.5h4a.75.75 0 0 0 0-1.5h-4Z" />,
+  'Sb_Filter'
+);
+
+const emptyFn = () => {
+  return;
+};
+
+export default {
+  title: '@ps/Unstable_Tag',
+  component: SbUnstable_Tag,
+  excludeStories: ['SbUnstable_Tag'],
+  parameters: {
+    actions: {
+      // override default actions regex
+      //  deleteIcon rendering is based on onDelete being supplied, so substitute
+      //  custom argType control below.
+      argTypesRegex: '^on(?!Delete).*',
+    },
+  },
+  argTypes: {
+    icon: {
+      control: 'select',
+      options: ['undefined', '$', 'Filter'],
```

Comment:
What is the '$' option here? it didn't do anything in storybook when I clicked it.

Reply:
Wasn't meant to be there, i copied the object from another story and forgot to remove that.